src/main/java/org/openelisglobal/common/provider/query/SampleSearchPopulateProvider.java

```diff
@@ -200,7 +200,7 @@ private void createReturnXML(Sample sample, String patientId, StringBuilder xml)
     private Organization getOrganizationForSample(Sample sample) {
         SampleOrganization so = new SampleOrganization();
         so.setSample(sample);
-        sampleOrganizationService.getDataBySample(so);
+        sampleOrganizationService.getDataBySample(sample);
```

Comment:
the original code seemed right. i dont see how the `so` object gets assigned the Organization object fetched from the DB.
unless you do `so = sampleOrganizationService.getDataBySample(sample);`

Reply:
The original code cause `null pointer exeception` while retrieving organization for sample. Because `so` does not keep the `sampleId`  from the sample object in parameter